js/src/Builder.js

```diff
@@ -25,6 +26,13 @@ var _ = require('underscore')
 var d3_select = require('d3-selection').select
 var d3_selection = require('d3-selection').selection
 var d3_json = require('d3-request').json
+var d3Interpolate = require("d3-interpolate")
```

Comment:
let's stick with _ separated variable names for now (i.e. `d3_interpolate`)

Eventually, I do want to move to camelCase, but we should do that all at once.